app/src/main/java/com/codepath/chefster/models/Ingredient.java

```diff
@@ -1,12 +1,14 @@
 package com.codepath.chefster.models;
 
+import android.os.Parcelable;
+
 import org.parceler.Parcel;
 
 /**
  * This class represents an ingredient that's a part of a dish with its price and amount needed
  */
 @Parcel
-public class Ingredient {
+public class Ingredient implements Parcelable{
```

Comment:
the annotation above (@Parcel) means we're using `Parceler` which does the implementation for you and you don't need to do this